datastream-server/src/main/java/com/linkedin/datastream/server/assignment/LoadBasedPartitionAssignmentStrategy.java

```diff
@@ -166,4 +176,29 @@ private ClusterThroughputInfo fetchPartitionThroughputInfo(DatastreamGroup datas
     }, Objects::nonNull, _throughputInfoFetchRetryPeriodMs, _throughputInfoFetchTimeoutMs)
         .orElseThrow(RetriesExhaustedException::new);
   }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public List<BrooklinMetricInfo> getMetricInfos() {
+    List<BrooklinMetricInfo> metricInfos = new ArrayList<>();
+    metricInfos.add(new BrooklinMeterInfo(CLASS_NAME + "." + THROUGHPUT_INFO_FETCH_RATE));
+    metricInfos.addAll(_assigner.getMetricInfos());
+    return Collections.unmodifiableList(metricInfos);
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public void cleanupStrategy() {
+    _assigner.cleanupMetrics();
+    super.cleanupStrategy();
+  }
+
+  @Override
+  protected void unregisterMetrics(String datastream) {
+    _assigner.unregisterMetricsForDatastream(datastream);
```

Comment:
This method still has to be here, because there's a path for cleaning metrics in `StickyPartitionAssignmentStrategy` which calls this method.